test/forms/AddTeamMembersToGroupFormSpec.scala

```diff
@@ -43,102 +46,121 @@ class AddTeamMembersToGroupFormSpec extends AnyWordSpec with Matchers with Guice
     "be fillable with a AddTeamMembersToGroup" in {
       val validatedForm = AddTeamMembersToGroupForm
         .form()
-        .fill(AddTeamMembersToGroup(hasAlreadySelected = false, None, Some(List(member1, member2))))
+        .fill(
+          AddTeamMembersToGroup(hasAlreadySelected = false,
+                                None,
+                                Some(List(member1, member2))))
       validatedForm.hasErrors shouldBe false
       validatedForm.value shouldBe Some(
-        AddTeamMembersToGroup(hasAlreadySelected = false, None, Some(List(member1, member2)))
+        AddTeamMembersToGroup(hasAlreadySelected = false,
+                              None,
+                              Some(List(member1, member2)))
       )
     }
 
     "be successful when button is Continue and team members are non-empty" in {
       val params = Map(
         hasAlreadySelected -> List("false"),
-        search             -> List.empty,
-        members            -> List(encode(member1), encode(member2))
+        search -> List.empty,
+        members -> List(encode(member1), encode(member2))
       )
       val boundForm = AddTeamMembersToGroupForm
         .form(ButtonSelect.Continue)
         .bindFromRequest(params)
       boundForm.value shouldBe Some(
-        AddTeamMembersToGroup(hasAlreadySelected = false, None, Some(List(member1, member2)))
+        AddTeamMembersToGroup(hasAlreadySelected = false,
+                              None,
+                              Some(List(member1, member2)))
       )
     }
 
     "have errors when team members is empty and hasAlreadySelected is false" in {
       val params = Map(
         hasAlreadySelected -> List("false"),
-        search             -> List.empty,
-        members            -> List.empty
+        search -> List.empty,
+        members -> List.empty
       )
       val boundForm = AddTeamMembersToGroupForm
         .form(ButtonSelect.Continue)
         .bindFromRequest(params)
-      boundForm.errors shouldBe List(FormError("members", List("error.select-members.empty")))
+      boundForm.errors shouldBe List(
+        FormError("", List("error.select-members.empty")))
```

Comment:
```
      case Continue =>
        val formWithMaybeGlobalError = Form(
          addTeamMembersToGroupMapping.verifying(emptyTeamMemberConstraint)
        )
        if (formWithMaybeGlobalError.hasErrors)
          formWithMaybeGlobalError.copy(errors = Seq(FormError("members", "error.select-members.empty")))
        else formWithMaybeGlobalError
```

Is is possible that the continue case could be throwing the error with and without the key? @max-hmrc 

I'm not seeing the non-deterministic behaviour locally but just an idea